### PR TITLE
Updated cross/around-boresight error calculation and added control error stats

### PR DESCRIPTION
**Pull request by galensavidge**

### CODE REVIEW

File: aces/sim/analysis/inlineAnalysis.py [hunk 1]
```diff
@@ -47,6 +47,57 @@
...
+    frame B. For example A could be the "true" B frame, or A could be a
+    reference attitude from guidance.
+
+    Assumptions:
+        1. The "true" or desired boresight lies on the A frame X-axis.
+        2. The around-boresight and cross-boresight angles are less than 90
+           degrees.
+
+    Args:
+        sigmaError: Some error MRP from B to A.
```
Comment (alboehm) [inline aces/sim/analysis/inlineAnalysis.py:63]: If the DCM is [BA], I think the MRP would reflect the attitude of B relative to A, so from A to B. please clarify

File: aces/sim/analysis/inlineAnalysis.py [hunk 2]
```diff
@@ -66,44 +117,21 @@ def checkPointingDeterminationCompliance(simTimes, dataSigmaEstError, dataModeId
                   period, radians.
     """
 
-    # Note to whoever ends up working on this in the future: this is probably not the correct way of determining
-    # pointing knowledge for L3-ATD01. However, the requirements as currently written don't really make much sense to me
-    # so this is what I implemented. In the future, this function should be reassessed/rewritten to compute whatever
-    # pointing knowledge really means.
+    # TODO: Note to whoever ends up working on this in the future: this is
+    # probably not the correct way of determining pointing knowledge for
+    # L3-ATD01. However, the requirements as currently written don't really
```
Comment (alboehm) [inline aces/sim/analysis/inlineAnalysis.py:122]: L3-ATD01 is that it will have 6 sun sensors. I think the requirements you want are L2-ATD05 and L2-ATC04.

**Review by alboehm — APPROVED** [review]

File: aces/sim/analysis/inlineAnalysis.py (hunk 1 above)
Comment (galensavidge) [reply aces/sim/analysis/inlineAnalysis.py:63]: Yeah that should say A to B, whoops

File: aces/sim/analysis/inlineAnalysis.py (hunk 2 above)
Comment (galensavidge) [reply aces/sim/analysis/inlineAnalysis.py:122]: All I personally added was the TODO part but I can update it now if you want

File: aces/sim/analysis/inlineAnalysis.py (hunk 1 above)
Comment (galensavidge) [reply aces/sim/analysis/inlineAnalysis.py]: Fixed

File: aces/sim/analysis/inlineAnalysis.py (hunk 2 above)
Comment (galensavidge) [reply aces/sim/analysis/inlineAnalysis.py]: Updated L3-ATD01 -> L2-ATD05

**Review by alboehm — APPROVED** [review]

**Pull request merged**